scripts/ci/run-vagrant-tests.sh

```diff
@@ -20,6 +20,15 @@ do
   DEPLOYMENT_MODE=$mode vagrant up --provision-with common
   DEPLOYMENT_MODE=$mode vagrant provision
   retcode=$?
+
+  # skydive testing
+  if [ $retcode -ne 0 ] ; then
```

Comment:
In this case, we probably want to compile the tests on the host, copy them to the guest and run them. This is done in https://github.com/skydive-project/skydive/blob/master/scripts/ci/run-kolla-tests.sh#L9 and https://github.com/skydive-project/skydive/blob/master/scripts/ci/kolla/Vagrantfile#L14

Reply:
I don't like so much not using the environment / vm that we designed to run for.
I mean we can do the trick to compile on the host and push to the vm, but in this case why working hard to setup the vm correctly with all the dev packages ready to build and run ?

Meanwhile I like the idea of Vagrantfile integrate the test script, that we can trig/run by setting an environment variable